migrations/V131__Add_Payment_Reonciliation_Frequency.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE plan_payments
+  ADD COLUMN payment_reconciliation_frequency_continually ZERO_STRING,
+  ADD COLUMN payment_reconciliation_frequency_other ZERO_STRING,
+  ADD COLUMN payment_reconciliation_frequency_note ZERO_STRING,
+  ADD COLUMN payment_reconciliation_frequency FREQUENCY_TYPE_NEW[] DEFAULT NULL;
```

Comment:
Side note here, this doesn't hurt anything, but you shouldn't need to add default NULL to this column definition. Since the column is nullable by default, this condition is implied.

Reply:
Thanks Patrick! And no need to either, this is still completely valid. I just wanted to call it out.